### PR TITLE
new parser for systemctl show httpd

### PR DESCRIPTION
**Pull request by TripleJQK**

### CODE REVIEW

**Comment by TripleJQK** [comment]
https://github.com/RedHatInsights/insights-frontend-assets/pull/53

File: insights/parsers/systemctl_show.py [hunk 1]
```diff
@@ -20,13 +20,17 @@
...
+-----------------------------------------------------
 """
 
+from insights.specs import Specs
+
 from .. import LegacyItemAccess
 from .. import Parser
 from .. import parser
 from ..parsers import split_kv_pairs
-from insights.specs import Specs
```
Comment (bfahr) [inline insights/parsers/systemctl_show.py]: @TripleJQK you could just change this line to `from ..specs import Specs`.  Its not clear why you moved it above the other imports.

File: insights/parsers/tests/test_systemctl_show.py [hunk 2]
```diff
@@ -74,7 +82,9 @@ def test_systemctl_show_doc_examples():
         'SystemctlShowPulpResourceManager': SystemctlShowPulpResourceManager,
         'systemctl_show_pulp_resource_manager': SystemctlShowPulpResourceManager(context_wrap(SYSTEMCTL_SHOW_EXAMPLES)),
         'SystemctlShowPulpCelerybeat': SystemctlShowPulpCelerybeat,
-        'systemctl_show_pulp_celerybeat': SystemctlShowPulpCelerybeat(context_wrap(SYSTEMCTL_SHOW_EXAMPLES))
+        'systemctl_show_pulp_celerybeat': SystemctlShowPulpCelerybeat(context_wrap(SYSTEMCTL_SHOW_EXAMPLES)),
+        'SystemctlShowHttpd': SystemctlShowHttpd,
```
Comment (bfahr) [inline insights/parsers/tests/test_systemctl_show.py]: Is this line needed?

File: insights/parsers/systemctl_show.py (hunk 1 above)
Comment (TripleJQK) [reply insights/parsers/systemctl_show.py]: modified

**Pull request merged**